dataprofiler/profilers/datetime_column_profile.py

```diff
@@ -163,6 +170,15 @@ def _validate_datetime(date, date_format):
 
         return converted_date
 
+    @staticmethod
```

Comment:
Do static methods such as this need to be covered in the [test_datetime_column_profile.py](https://github.com/capitalone/DataProfiler/pull/458/files#diff-9bba6fcc721a948265b1158d5a65dc4cd34a55984ff110d83782d9da29abc6b1).

Reply:
I think it's covered by the other functions output, with the `th`, but we should have an individual test with each day_suffix in a dataset being profiled to validate each rather than updating existing tests.